MediaAdminBundle/DependencyInjection/Compiler/RoleCompilerPass.php

```diff
@@ -29,5 +29,13 @@ public function process(ContainerBuilder $container)
             TreeFolderPanelStrategy::ROLE_ACCESS_UPDATE_MEDIA,
             TreeFolderPanelStrategy::ROLE_ACCESS_DELETE_MEDIA,
         ));
+
+        if ($container->hasParameter('open_orchestra_backoffice.role')) {
+            $param = $container->getParameter('open_orchestra_backoffice.role');
```

Comment:
Instead of ``$param`` can you find something more explicit like ``roles`` or ``rolesList`` or something like this ?